### DISCUSSION TITLE
Copy existing project within qfield cloud

### DISCUSSION THREAD

**Discussion by Firefly86** [opengisch/QField#3111] [Ideas]
I would love to copy an existing project within qfield cloud.

I created several project templates containing all layers. basemaps and settings we need.. Sometimes I need to create a new project based on these templates in the field. So no access to Qgis to create a new project and upload it to qfield cloud. A copy and rename function would help our workflow massivly

**Comment by suricactus** [comment]
While this is an enhancement that is on our TODO list, there is no immediate plan to implement it. If you want to speed up the development for such feature, drop us an email at info [at] opengis.ch.

**Comment by ZsanettMed** [comment]
Hello @Firefly86 ,

I added your request to our [QField Idea Platform](https://ideas.qfield.org/qfieldcloud-feature-requests/p/project-duplication-in-qfieldcloud) for community voting. Feel free to support it there. This conversation is now concluded. This thread will now be closed.

Best,
Zsanett
